### PR TITLE
[Notifi V4] Add "Clear all" feature to clean notification history 

### PR DESCRIPTION
**Pull request by eric-notifi**
## Background
Add new feature "Clear all" to delete all notification history

Screenshot 2023-09-21 at 11 22 18

### 9/28
- Rebase
- Update localization/ NotifiContext

### 9/27 
- Rebase and fix conflict

### 9/21
- Init new feature "Clear all"

### CODE REVIEW

File: packages/web/integrations/notifi/notifi-context.tsx [hunk 1]
```diff
@@ -75,18 +77,20 @@ const NotifiContextProviderImpl: FunctionComponent<PropsWithChildren<{}>> =
...
-            {children}
-          </NotifiModalContextProvider>
-        </NotifiConfigContext>
+        <NotifiLocalStorageProvider>
+          <NotifiConfigContext
+            type="SUBSCRIPTION_CARD"
+            id={
+              IS_TESTNET
+                ? "c5fb30811f5b47b79dd9a400480c2670"
+                : "c5fb30811f5b47b79dd9a400480c2670"
```
Comment (jonator) [inline packages/web/integrations/notifi/notifi-context.tsx:86]: Heads up these appear to be the same

File: packages/web/localizations/zh-hk.json
Comment (jonator) [inline packages/web/localizations/zh-hk.json:1]: Should we be including these changes to all of the localization files?

File: packages/web/integrations/notifi/notifi-context.tsx (hunk 1 above)
Comment (eric-notifi) [reply packages/web/integrations/notifi/notifi-context.tsx:86]: Yah, both Dev and Prod share the same cardId. If it is confusing, I will remove the condition check

File: packages/web/localizations/zh-hk.json
Comment (eric-notifi) [reply packages/web/localizations/zh-hk.json:1]: I am not sure why, but it does not translate the newly added field after executing `machine-translate`. Below is the output FYI. Is it familiar to you?
```bash
❯ yarn machine-translate
yarn run v1.22.19
$ inlang machine translate && yarn fix-machine-translate

 WARN  Machine translations are not very accurate. We advise you to only use machine translations in a build step to have them in production but not commit them to your repository. You can use the force flag (-f, --force) to skip this prompt in a build step.

Are you sure you want to machine translate? (y/n) y
ℹ 🌏 Found 12 languages with a total of 6793 messages.                                                                                          09:26:19
ℹ 📝 Translating to 11 languages. [es, fa, fr, ko, pl, pt-br, ro, tr, zh-cn, zh-hk, zh-tw]                                                      09:26:19

 ERROR  (intermediate value) is not iterable                                                                                                    09:26:20

  at Command2.translateCommandAction (node_modules/@inlang/cli/dist/main.js:70871:42)
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

$ node fix-machine-translate.js
Fixed JSON file: /Users/macbookpro4eric/Projects/notifi/osmosis-frontend/packages/web/localizations/en.json
Fixed JSON file: /Users/macbookpro4eric/Projects/notifi/osmosis-frontend/packages/web/localizations/es.json
Fixed JSON file: /Users/macbookpro4eric/Projects/notifi/osmosis-frontend/packages/web/localizations/fa.json
Fixed JSON file: /Users/macbookpro4eric/Projects/notifi/osmosis-frontend/packages/web/localizations/fr.json
Fixed JSON file: /Users/macbookpro4eric/Projects/notifi/osmosis-frontend/packages/web/localizations/ko.json
Fixed JSON file: /Users/macbookpro4eric/Projects/notifi/osmosis-frontend/packages/web/localizations/pl.json
Fixed JSON file: /Users/macbookpro4eric/Projects/notifi/osmosis-frontend/packages/web/localizations/pt-br.json
Fixed JSON file: /Users/macbookpro4eric/Projects/notifi/osmosis-frontend/packages/web/localizations/ro.json
Fixed JSON file: /Users/macbookpro4eric/Projects/notifi/osmosis-frontend/packages/web/localizations/tr.json
Fixed JSON file: /Users/macbookpro4eric/Projects/notifi/osmosis-frontend/packages/web/localizations/zh-cn.json
Fixed JSON file: /Users/macbookpro4eric/Projects/notifi/osmosis-frontend/packages/web/localizations/zh-hk.json
Fixed JSON file: /Users/macbookpro4eric/Projects/notifi/osmosis-frontend/packages/web/localizations/zh-tw.json
✨  Done in 4.97s.
```
Screen video also attached herewith FYI.

Comment (eric-notifi) [reply packages/web/localizations/zh-hk.json:1]: Yup! done.

**Review by jonator — APPROVED** [review]

**Pull request merged**